pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardfull/_generated_export_monitoringdashboardfull.golden

```diff
@@ -0,0 +1,48 @@
+apiVersion: monitoring.cnrm.cloud.google.com/v1beta1
+kind: MonitoringDashboard
+metadata:
+  name: monitoringdashboard-${uniqueId}
+spec:
+  columnLayout:
+    columns:
+    - weight: 2
+      widgets:
+      - title: Widget 1
+        xyChart:
+          dataSets:
+          - plotType: LINE
+            timeSeriesQuery:
+              timeSeriesFilter:
+                aggregation:
+                  perSeriesAligner: ALIGN_RATE
+                filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+              unitOverride: "1"
+          timeshiftDuration: 600.5s
+          yAxis:
+            label: y1Axis
+            scale: LINEAR
+      - text:
+          content: Widget 2
+          format: MARKDOWN
+      - title: Widget 3
+        xyChart:
+          dataSets:
+          - plotType: STACKED_BAR
+            timeSeriesQuery:
+              timeSeriesFilter:
+                aggregation:
+                  perSeriesAligner: ALIGN_RATE
+                filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+              unitOverride: "1"
+          yAxis:
+            label: y1Axis
+            scale: LINEAR
+      - logsPanel:
+          filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+          resourceNames:
+          - external: projects/${projectId}
+            kind: Project
+        title: Widget 4
+  displayName: monitoringdashboard-full
+  projectRef:
+    external: ${projectId}
```

Comment:
Same question. Also, these files does not have the yaml extension. Is that intended?

Reply:
I think the naming of these files could be improved, but this is the name that we're currently using.  No objection to changing it, but I think it should be another PR.

For example, we should probably:

* Make the extension .yaml
* Name it after the kind and name of what we are exporting
* Consider exporting more than the "primary" object